JK Hera/src/main/java/ayds/hera/songinfo/home/view/SongToReleaseDateStringMapper.kt

```diff
@@ -0,0 +1,39 @@
+package ayds.hera.songinfo.home.view
+
+import ayds.hera.songinfo.home.model.entities.*
+import java.time.Month
+
+interface SongToReleaseDateStringMapper {
+    fun map():String
+}
+
+internal class SongToDayReleaseDateStringMapper(private val song:Song) : SongToReleaseDateStringMapper {
+    override fun map():String {
+        val componentDate = song.releaseDate.split("-")
+        return "${componentDate[2]}/${componentDate[1]}/${componentDate[0]}"
```

Comment:
Acceder a los elementos del array es poco clean, hay que hacer un mapeo mental. Mejor agregar variables intermedias para que sea más claro:
```suggestion
        val year = componentDate[0]
        val month = componentDate[1]
        val day =  componentDate[2]
        return "$day/$month/$year"
```
Un detalle, podemos confiar en el factory y sabemos que siempre vamos a tener 3 elementos. En otro caso, faltaría el checkeo de cantidad de elementos para evitar una excepción en ejecución.